appinfo/info.xml

```diff
@@ -11,6 +11,10 @@
 	<namespace>LogReader</namespace>
 	<default_enable/>
 
+	<types>
+		<logging/>
+	</types>
```

Comment:
Does that mean the application will be loaded on all requests while it was not before?
Should we have a "console" type to only be loaded by occ?

Reply:
Yes, but it shouldn't have any significant impact, as the app is very light weight if you don't trigger it